configs/bcm72604B0_wpe_ml_defconfig

```diff
@@ -0,0 +1,106 @@
+BR2_arm=y
+BR2_cortex_a15=y
+BR2_ARM_FPU_NEON_VFPV4=y
+BR2_ARM_INSTRUCTIONS_THUMB2=y
+BR2_CCACHE=y
+BR2_OPTIMIZE_2=y
+BR2_TOOLCHAIN_BUILDROOT_GLIBC=y
+BR2_PACKAGE_HOST_LINUX_HEADERS_CUSTOM_4_9=y
+BR2_GLIBC_VERSION_2_23=y
+BR2_GCC_VERSION_6_X=y
+BR2_TOOLCHAIN_BUILDROOT_CXX=y
+BR2_TARGET_GENERIC_HOSTNAME="WPE"
+BR2_TARGET_GENERIC_ISSUE="Welcome to BCM72604"
+BR2_TARGET_GENERIC_CABUNDLE=y
+# BR2_TARGET_GENERIC_NETWORK is not set
+BR2_ROOTFS_DEVICE_CREATION_DYNAMIC_EUDEV=y
+BR2_TARGET_GENERIC_ROOT_PASSWD="root"
+BR2_TARGET_GENERIC_GETTY_PORT="ttyS0"
+BR2_SYSTEM_DHCP="eth0"
+BR2_ROOTFS_POST_BUILD_SCRIPT="board/kaon/post-build.sh"
```

Comment:
If we're relying on board specifics from kaon, shouldn't the defconfig be more specific to that board? eg. kaon72604B0_wpe_ml_defconfig?

Reply:
Good point. defconfig name has been renamed.